san/get.py

```diff
@@ -1,12 +1,26 @@
 import san.sanbase_graphql
+import operator
 from san.graphql import execute_gql
 from san.pandas_utils import convert_to_datetime_idx_df
 from san.query import get_gql_query, parse_dataset
+from functools import reduce
 
+def shorten_path(root, items):
+    return reduce(operator.getitem, items, root)
+    """
+    The first parameter (root) is the result from the execution of the query, the second is the list of keys,
+    the first is the first key in the result of the query, the last - the key that we need the data from.
+
+    The returned value is a shortened dictionary, from which the query can easily return the needed data.
+    """
 CUSTOM_QUERIES = {
     'ohlcv': 'get_ohlcv',
 }
 
+QUERY_PATH_MAP = {
+    'eth_top_transactions': ['query_0', 'ethTopTransactions']
```

Comment:
why `query_0`? We should have the automatic assignment of the numbers, right? If so, this query won't be always be with the `0` index. You can test by using `batch` and batching multiple `ethTopTransactions` -  you can add this as a test